Cargo.toml

```diff
@@ -11,10 +11,13 @@ repository = "https://github.com/myfreeweb/systemstat"
 
 [dependencies]
 time = "0.1"
-chrono = "0.4"
+chrono = {version = "0.4.19", features = ["serde"] }
 lazy_static = "1.0"
-bytesize = "1.0"
+bytesize = { version = "1.0.1", features = ["serde"] }
 libc = "0.2"
+serde = {version = "1.0.126", features = ["derive"] }
```

Comment:
Any reason to use at least that specific version rather than `1.0`? I generally specify only the major version.

Reply:
Sorry my vscode extension did that automatically.